### PR TITLE
Verify email email templates DIN-10

### PR DESCRIPTION
**Pull request by jasongrier**

### CODE REVIEW

File: email-templates/danish_templates/verify_email.xml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        letter-spacing: -0.0026em;
+        color: #fff;
+        border: 0;
+        border-radius: 6px;
+        text-decoration: none;
+        cursor: pointer;
+      }
+    </mj-style>
+
+    <mj-preview>INSERT PREVIEW TEXT HERE, AT LEAST 90 CARACHTERS LONG INSERT PREVIEW TEXT HERE, AT LEAST 90 CARACHTERS LONG</mj-preview>
```
Comment (theilgaard) [inline email-templates/danish_templates/verify_email.xml:49]: Missing copy here.

File: email-templates/verify_email.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
+<mjml>
+  <mj-head>
+    <mj-title>INSERT TITLE HERE</mj-title>
```
Comment (theilgaard) [inline email-templates/verify_email.html:3]: Missing title copy

File: email-templates/verify_email.html [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        letter-spacing: -0.0026em;
+        color: #fff;
+        border: 0;
+        border-radius: 6px;
+        text-decoration: none;
+        cursor: pointer;
+      }
+    </mj-style>
+
+    <mj-preview>INSERT PREVIEW TEXT HERE, AT LEAST 90 CARACHTERS LONG INSERT PREVIEW TEXT HERE, AT LEAST 90 CARACHTERS LONG</mj-preview>
```
Comment (theilgaard) [inline email-templates/verify_email.html:49]: Missing preview copy

**Comment by theilgaard** [comment]
@jasongrier is this done?

**Comment by jasongrier** [comment]
Done. But I don't have permission to merge. @theilgaard

**Comment by theilgaard** [comment]
> Done. But I don't have permission to merge. @theilgaard

Roger. I just noticed a slight error, the verification URL should be parameterized. You can use `{{ url_verify_email }}`

**Comment by jasongrier** [comment]
> > Done. But I don't have permission to merge. @theilgaard
> 
> Roger. I just noticed a slight error, the verification URL should be parameterized. You can use `{{ url_verify_email }}`

Fixed that; still can't merge though

**Pull request merged**